src/test/java/kitchenpos/application/MenuServiceTest.java

```diff
@@ -0,0 +1,145 @@
+package kitchenpos.application;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import java.util.Optional;
+import kitchenpos.dao.MenuDao;
+import kitchenpos.dao.MenuGroupDao;
+import kitchenpos.dao.MenuProductDao;
+import kitchenpos.dao.ProductDao;
+import kitchenpos.domain.Menu;
+import kitchenpos.domain.MenuProduct;
+import kitchenpos.generator.MenuGenerator;
+import kitchenpos.generator.ProductGenerator;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+
+public class MenuServiceTest extends ServiceTest {
+
+    @Mock
+    private MenuDao menuDao;
+
+    @Mock
+    private MenuGroupDao menuGroupDao;
+
+    @Mock
+    private MenuProductDao menuProductDao;
+
+    @Mock
+    private ProductDao productDao;
+
+    @InjectMocks
+    private MenuService menuService;
+
+    @DisplayName("메뉴 등록")
+    @Test
+    void create() {
+        Menu menuToCreate = MenuGenerator.newInstance("후라이드+후라이드", 19000, 1L, Collections.singletonList(MenuGenerator.newMenuProduct(1L, 2)));
+        when(menuDao.save(menuToCreate)).thenAnswer(invocation -> {
+            Menu menu = invocation.getArgument(0);
+            return MenuGenerator.newInstance(2L, menu.getName(), menu.getPrice().intValue(), menu.getMenuGroupId(), menu.getMenuProducts());
+        });
+        when(menuProductDao.save(any(MenuProduct.class))).thenAnswer(invocation -> {
+            MenuProduct menuProduct = invocation.getArgument(0);
+            return MenuGenerator.newMenuProduct(3L, menuProduct.getMenuId(), menuProduct.getProductId(), (int) menuProduct.getQuantity());
+        });
+        when(menuGroupDao.existsById(menuToCreate.getMenuGroupId())).thenReturn(true);
+        when(productDao.findById(1L)).thenAnswer(invocation -> Optional.of(ProductGenerator.newInstance(1L, "후라이드", 16000)));
+
+        Menu actual = menuService.create(menuToCreate);
+
+        Menu expected = new Menu();
+        expected.setId(2L);
+        expected.setName(menuToCreate.getName());
+        expected.setPrice(menuToCreate.getPrice());
+        expected.setMenuGroupId(1L);
+        expected.setMenuProducts(Collections.singletonList(MenuGenerator.newMenuProduct(3L, 2L, 1L, 2)));
+        verify(menuDao, times(1)).save(menuToCreate);
```

Comment:
```suggestion
        expected.setMenuProducts(Collections.singletonList(MenuGenerator.newMenuProduct(3L, 2L, 1L, 2)));
        
        verify(menuDao, times(1)).save(menuToCreate);
```

// then
에 해당하는 부분은 띄워주시는 게 어떨까요?

given, when, then은 취향이지만, 맥락이 바뀌는 부분에서는 의미있는 공백이 있으면 좋겠다는 생각이 들었습니다🙂.

Reply:
의도해서 띄운다고했는데 실수로 빼먹었네요 😅 수정하겠습니다!